apps/wallet/src/ui/app/shared/transaction-summary/cards/BalanceChanges.tsx

```diff
@@ -45,6 +76,53 @@ function BalanceChangeEntry({ change }: { change: BalanceChange }) {
 	);
 }
 
+function BalanceChangeEntries({ changes }: { changes: BalanceChange[] }) {
+	const recognizedPackagesList = useRecognizedPackages();
+	const normalizedRecognizedPackages = useMemo(
+		() => recognizedPackagesList.map((itm) => normalizeSuiObjectId(itm)),
+		[recognizedPackagesList],
+	);
+	const { recognizedTokenChanges, unRecognizedTokenChanges } = useMemo(() => {
```

Comment:
Rather than duplicating this logic, can you make this part of the balance change construction in the core?

Reply:
This is done.  moved the logic to getBalanceChangeSummary